lib/dry/validation/result.rb

```diff
@@ -81,3 +81,11 @@ def result_ast
     end
   end
 end
+
+begin
```

Comment:
dry-t shouldn't depend on dry-m so we need this feature there too...dry-core? :D